### DISCUSSION TITLE
I need a Remote Job - Self Taught. Learn more below

### DISCUSSION THREAD

**Discussion by hay1ando** [remoteintech/remote-jobs#1154] [Q&A] [answered]
Good evening community, I'm 25 years old, I want to change my work life to remote position and also to acquire a position in a company where my work and or our groups efforts make a difference and we help move processes smoothly and effectively. I'm currently self employed and that's basically how it's been all my life. I have never applied to any jobs and or held any positions. I have done great for myself. But now to how I got here and why I belong in IT and my dream to get into the aero space industry and it industry. 
My life has been outstanding and have seen and Been around many life learning experiences since I was a young baby. When my parents left me and my brother in our country for a few years until they were able to figure a few things out financially here in the states. I was always outside and I feel at a young age this made me adventurous and to try new things. Also to keep my mind open. First time I seen a computer at age 6 my cousins house and he asked me to turn the computer off which I told him "no problem, I got this " and continued to turn the computer off by pressing the power button on the desktop and the monitor on the same time. Right away my cousin got upset but hey at least I tried even tho I failed, miserably.
At age 10 me and my brother actually discovered Bitcoin which back then was $5. We started mining as babies. That didn't go too far, we got shut down after our father wouldn't agree to give us a $1000 at age 10 to invest into something they didn't understand. It's ok that we didn't make millions. I continued to be online and got involved with opening my first phone repair company which I had to design a website and start marketing to build my business. At age 13 I started making websites and fixing tech devices. But marketing and building online became a thing and I sat by the computer day and night learning and applying. Overtime things started to fade in regards to so many different learning opportunities and schools but I was handling situations better on my own and learning in school for me was not a big deal. Things started to fade and I joined my dads contracting trade and started building houses. Because no one in my life or around me was actually in this sphere. I was totally alone and needed cash. I chose HVAC in air conditioning and built a multi million dollar HVAC service business. Few years back I regained my confidence to gain more clients in building websites and strategy for local businesses. But I have so many good ideas but people who I hired overtime were not as effective as I needed them to be and things failed. Now at 25 my gf is pregnant and I need a stable remote job where I can be with my family and be available online to service the companies needs. 

I hope my story was funny, even if u can't help I appreciate you listening and hopefully you can direct me to through the right path.

**Comment by Cephas12** [comment]
The best way to search for opportunities is the web. Don't give up, you will hit your target pretty soon 🔜 💕

**Comment by dougaitken** [comment, marked as answer]
Hey @hay1ando my advice would be to focus on what you want to do and what you're good at. Try the HVAC business again, stay small but aim for profit. Otherwise, find a local job that will pay the bills while you focus on gaining new skills. 

Although this repository is related to remote jobs, it is only a listing, there isn't a service to get a job directly through the site. Good luck with your search.

**Comment by douglasdcm** [comment]
Hi @hay1ando I hope you have found a new job. I'm working on a project to collect and disclose job positions. The web site is available in [here](https://vagaspramim.onrender.com), so you can search past your resumé or just type for "remote" to get the positions that match the query. The [source code](https://github.com/douglasdcm/crawler_of_positions) is free for any one who wants to contribute.

I'm going to include some companies from this repository there. Congratulations for the maintainers, by the way!
